projects/wave-core-new/src/lib/layers/layer-list/layer-list-element/layer-list-element.component.ts

```diff
@@ -151,4 +157,42 @@ export class LayerListElementComponent implements OnDestroy, OnChanges {
         this.clipboard.copy(layer.workflowId);
         this.notificationService.info('Copied workflow id to clipboard');
     }
+
+    downloadMetadata(layer: Layer): void {
+        this.notificationService.info(`Downloading metadata for layer ${layer.name}`);
+
+        this.userService
+            .getSessionTokenForRequest()
+            .pipe(
+                mergeMap((token) => this.backend.downloadWorkflowMetadata(layer.workflowId, token)),
+                tap((event) => {
+                    if (event.type !== HttpEventType.DownloadProgress) {
+                        return;
+                    }
+
+                    const fraction = event.total ? event.loaded / event.total : 1;
+                    this.notificationService.info(`Metadata download: ${100 * fraction}%`);
+                }),
+                last(),
+            )
+            .subscribe({
+                next: (event) => {
+                    if (event.type !== HttpEventType.Response || event.body === null) {
+                        return;
+                    }
+
+                    const zipArchive = new File([event.body], filenameFromHttpHeaders(event.headers) ?? 'metadata.json');
```

Comment:
why would you call a zipFile metadata.json?

Reply:
[4d7cd45](https://github.com/geo-engine/geoengine-ui/pull/388/commits/4d7cd4539baaa1d751d6d9c168d8f77f91e994c1)